modules/installation-osp-kuryr-port-pools.adoc

```diff
@@ -23,3 +23,8 @@ Prior to installing a cluster, you can set the following parameters in the `clus
 +
 If your OpenStack port quota is low, or you have a limited number of IP addresses on the pod network, consider setting this option to ensure that unneeded ports are deleted.
 * The `poolBatchPorts` parameter defines the maximum number of Neutron ports that can be created at once. The default value is `3`.
+
+[IMPORTANT]
+====
+Red Hat does not provide guidance on defining a maximum number of Neutron ports but suggests setting a value to avoid the issue of having a variety of Bulk Ports requests, that is, having multiple pods created simultaneously.
```

Comment:
How about:
Red Hat does not provide guidance on defining the number of bulk ports setting. But, highlights a known issue
that happens when multiple Ports are created in bulk at the same time, that is multiple Pods created simultaneously.

and then we can link the issue?

Reply:
@MaysaMacedo From what I understand, e try to avoid documenting bug references outside of the release notes because it is difficult to make sure that it all gets cleaned up once the bug is fixed. 

I have included the link to the bug as a known issue in my release notes - https://github.com/openshift/openshift-docs/pull/46521

Reply:
@MaysaMacedo Yes, that makes sense given the circumstances and nature of the known issue. I am currently working on the updates for this PR and not the release notes. I wanted to confirm, is there a planned fixed for this issue in a later version or will it continue to possible affect all versions?

Reply:
The fix was already merged in OpenStack and there is an errata. So, I think we can leave this note to the release notes instead to avoid the cleaning up.

Reply:
@MaysaMacedo I have added this to the Release Notes PR: https://github.com/openshift/openshift-docs/pull/46521

I am also wondering if we should still leave a note for suggesting users to set a value in the documentation without linking back to the actual bug?